peachjam/models/judgment.py

```diff
@@ -344,6 +359,13 @@ def pre_save(self):
         self.assign_title()
         super().pre_save()
 
+    def save(self, *args, **kwargs):
+        # ensure judgment locality matches the court's locality
+        if self.court.locality:
```

Comment:
Do this in `pre_save` because it changes the MNC. 

Also force the country if it is set.